### DISCUSSION TITLE
Exclude list item attachments with Query template

### DISCUSSION THREAD

**Discussion by Terenyeitamas** [microsoft-search/pnp-modern-search#3330] [Q&A]
Hello!

How can I exclude list item attachments with Query template?
I have a list: _https://***.sharepoint.com/sites/***/Lists/NewsApp/*_
The attachments are stored here: _https://***.sharepoint.com/sites/***/Lists/NewsApp/Attachments/*/*_

My Query template:

_{searchTerms}*
Path:"https://***.sharepoint.com/sites/***/*"
NOT Path:"https://***.sharepoint.com/sites/***/Lists/NewsApp/Attachments/*/*"_

But it does not work. I see attachments in the search results.

I see if "_LinkOfficeChild_" is not null, there is an attachment. Can I exclude "_LinkOfficeChild is not null_" from the query? 

Thanks,
Tamas

**Comment by kasperbolarsen** [comment]
like this?
{searchTerms}
path:https://XXX.sharepoint.com/sites/PnPModernSearch/Lists/ListWithAttachments* 
-Path:https://XXX.sharepoint.com/sites/PnPModernSearch/Lists/ListWithAttachments/Attachments*

**Reply by Terenyeitamas** [reply]
No, it does not work, but I find the solution.

I mapped the LINK.OFFICECHILD managed property to RefinableString114. After that I added _-RefinableString114:"http*"_ to the Query template. 

This works for me.

Best,
Tamas

**Comment by Terenyeitamas** [comment]
I mapped the LINK.OFFICECHILD managed property to RefinableString114. After that I added -RefinableString114:"http*" to the Query template.

This works for me.
